polaris/dataset/_dataset.py

```diff
@@ -138,12 +137,11 @@ def load_zarr_root_from_hub(self):
         """
         Loads a Zarr archive from the Hub.
         """
-        from polaris.hub.client import PolarisHubClient
         from polaris.hub.storage import StorageSession
+        import zarr
 
-        with PolarisHubClient() as client:
-            with StorageSession(client, "read", self.urn) as storage:
-                return zarr.open_consolidated(store=storage.store("extension"))
+        store = StorageSession.store(self)
```

Comment:
Related to my other comment about the `store` method, I would change this to:

```suggestion
        store = fsspec.get_mapper(self.zarr_root_path)
```